index.html

```diff
@@ -14,6 +16,83 @@
 </head>
 
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+  <!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+
+  <div class="container">
+    <header>
+      <nav>
+        <a href="">Services</a>
+        <a href="">Product</a>
+        <a href="">Vision</a>
+        <a href="">Features</a>
+        <a href="">About</a>
+        <a href="">Contact</a>
+      </nav>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </header>
+
+    <section>
+      <div class="left">
+        <h1>
+          Innovation
+          On
+          Demand
+        </h1>
+        
+        <div class="button">
+          <a href="#">Get Started</a>
+        </div>
+      </div>
+
+      <div class="right">
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+    </section>
+
+    <section class="top-content">
+      <h5>Features</h5>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+
+      <h5>About</h5>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+
```

Comment:
```suggestion
    <section class="top-content">
      <h5>Features</h5>
      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
       <h5>About</h5>
      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
    </section>
``` you would want to contain both h5 and p tags within a div